x-pack/test/licensing_plugin/public/updates.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../services';
+import { LicensingPluginSetup } from '../../../plugins/licensing/public';
+import { createScenario } from '../scenario';
+import '../../../../test/plugin_functional/plugins/core_provider_plugin/types';
+
+// eslint-disable-next-line import/no-default-export
+export default function(ftrContext: FtrProviderContext) {
+  const { getService } = ftrContext;
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+
+  const scenario = createScenario(ftrContext);
+
+  describe('changes in license types', () => {
+    after(async () => {
+      await scenario.startBasic();
+    });
+
+    it('provides changes in license types', async () => {
+      await scenario.setup();
+      await scenario.waitForPluginToDetectLicenseUpdate();
+
+      expect(
+        await browser.executeAsync(async (cb: Function) => {
+          // this call enforces signature check to detect license update
+          // and causes license re-fetch
+          await window.np.setup.core.http.get('/');
+          await window.np.testUtils.delay(100);
```

Comment:
Question: What is the upside of exposing that kind of static utils from the globally exposed `np` instead of static import?

Reply:
Do you mean this?
```js
import { delay } from '...';

await browser.executeAsync(async (cb: Function) => {
  await window.np.setup.core.http.get('/');
  await delay(100);
  ...
});
```
Selenium serializes a function passed into `browser.executeAsync(fn)` to run it in a browser. Any closures will be lost.

Reply:
Oh yea right. That's indeed an excellent answer.